## Hanna's Midterm
Data on more than 5,800 clergy who have been listed as credibly accused of sexual abuse in reports released by Catholic dioceses and religious orders from 2002-2020. 

### Import Python tools

In [1]:
import pandas as pd

### Read data

In [2]:
url = 'file:///Users/hannakang/Downloads/clergymembers.html'

In [3]:
df = pd.read_html(url)[0]

In [4]:
df.rename(columns=df.iloc[0], inplace = True)
df.drop([0], inplace = True)

### Basic information about data

In [5]:
df

id              name first_name  last_name  \
1     652690      Gerald Prinz     Gerald      Prinz   
2     652691     Gerard Kinane     Gerard     Kinane   
3     652692   Lawrence Cavell   Lawrence     Cavell   
4     652693    Patrick Kujawa    Patrick     Kujawa   
5     652694        Ramon Luce      Ramon       Luce   
...      ...               ...        ...        ...   
6749  652685  Claude Boudreaux     Claude  Boudreaux   
6750  652686        Dac Nguyen        Dac     Nguyen   
6751  652687       Dale Guidry       Dale     Guidry   
6752  652688      Daniel Poche     Daniel      Poche   
6753  652689   Etienne LeBlanc    Etienne    LeBlanc   

                                                 status birth_date ord_date  \
1     All priestly faculties permanently removed by ...        NaN     1968   
2     All priestly faculties permanently removed wit...        NaN     1973   
3     All priestly faculties permanently removed by ...        NaN     1989   
4                  Convicted and currently incarcerated        NaN     1996   
5                       Voluntarily left the priesthood        NaN     1995   
...                                                 ...        ...      ...   
6749                                           Deceased        NaN     1955   
6750                 Current status or location unknown        NaN     1988   
6751                                           Deceased        NaN     1982   
6752  All priestly faculties permanently removed by ...        NaN     1978   
6753  All priestly faculties permanently removed by ...        NaN     1971   

                                            assignments  \
1             Served at Saint Francis de Sales in Houma   
2                     Served at Saint Hilary in Mathews   
3                   Served at Holy Cross in Morgan City   
4     Served at Holy Cross in Morgan City and Saint ...   
5                     Served at Holy Savior in Lockport   
...                                                 ...   
6749                                                NaN   
6750        Served at Thanh Gia (Holy Family) in Amelia   
6751  Served at Holy Cross in Morgan City and Saint ...   
6752                Served at Holy Cross in Morgan City   
6753                      Served at Annunziata in Houma   

                     diocese_id                     diocese_name  \
1        Saint Gregory in Houma    and Saint Louis in Bayou Blue   
2                          2581       Diocese of Houma-Thibodaux   
3      Sacred Heart in Montegut  Saint Francis de Sales in Houma   
4                          2581       Diocese of Houma-Thibodaux   
5                          2581       Diocese of Houma-Thibodaux   
...                         ...                              ...   
6749                       2581       Diocese of Houma-Thibodaux   
6750                       2581       Diocese of Houma-Thibodaux   
6751                       2581       Diocese of Houma-Thibodaux   
6752    Sacred Heart in Cut Off        Saint Bernadette in Houma   
6753  Holy Cross in Morgan City             Holy Family in Dulac   

                                              list_name  \
1                                                  2581   
2     Disclosure of Names - The following priests ar...   
3                          Saint Genevieve in Thibodaux   
4     Disclosure of Names - The following priests ad...   
5     Disclosure of Names - The following priests ar...   
...                                                 ...   
6749  Disclosure of Names - The following priests ar...   
6750  Disclosure of Names - The following priests ar...   
6751  Disclosure of Names - The following priests ad...   
6752                    Saint Francis de Sales in Houma   
6753                              Saint Eloi in Theriot   

                                                   slug  \
1                            Diocese of Houma-Thibodaux   
2     ranging from inappropriate ph

### How many unique counts of sexual assault were reported?

In [6]:
len(df)

6753

### What is the distribution?

In [7]:
df.describe()

id           name first_name last_name    status birth_date  \
count     6753           6753       6752      6718      6109       1893   
unique    6753           5897        796      4383      1161        297   
top     652319  Robert Poandl       John    Murphy  Deceased       1946   
freq         1             10        576        27      2576         49   

       ord_date       assignments diocese_id                diocese_name  \
count      3506              3985       6686                        6741   
unique      146              2269       2765                        2826   
top        1955  Assistant Pastor       2496  Archdiocese of Los Angeles   
freq         97                99        301                         301   

                                                list_name  \
count                                                6753   
unique                                               2833   
top     CONSOLIDATED LIST OF PERSONS NAMED IN THE REPO...   
freq                                                  297   

                                           slug pp_id  
count                                      6751  6750  
unique                                     5038  5238  
top     WITH CURRENT STATUS THROUGH NOVEMBER 30  2018  
freq                                        297   297

### when did the clergy get ordinated? (oldest)

In [8]:
df.sort_values('ord_date', ascending=True).head()

id                       name first_name           last_name  \
4485   70998           Frederick George  Frederick              George   
4091   70466       Joseph Maxime Mooney     Joseph       Maxime Mooney   
5653   65865           Joseph T. Neeson     Joseph              Neeson   
2293  652166          Joseph Thibaudeau     Joseph          Thibaudeau   
5695   65935  Pierre Celestin Cambiaire     Pierre  Celestin Cambiaire   

                           status      birth_date ord_date  \
4485  Left religious order (1998)       Dismissed   (2004)   
4091              Deceased (1965)  Retired (1956)     1886   
5653                     Deceased             NaN     1890   
2293                     Deceased            1869     1896   
5695                     Deceased            1871     1898   

                assignments                   diocese_id  \
4485                    NaN                         1975   
4091                   1916                St. Johnsbury   
5653          December 1890    Associate Pastor/Chaplain   
2293             St. Jean's  Duluth - 6/30/1910 - ?/1943   
5695  Sacred Heart of Jesus          Cameron; St. George   

                               diocese_name               list_name  \
4485  St. Michael Catholic Church (Gastonia                   N.C.)   
4091                             Notre Dame             (Assistant)   
5653                       Associate Pastor  Saint Alphonsus Parish   
2293                                   2560       Diocese of Duluth   
5695                Baton Rouge; St. Joseph   Grosse-Tete; St. Leon   

                                                   slug  \
4485                                               2548   
4091                                1917-22; Burlington   
5653                                     Daviess County   
2293  Clergy with credible claims against them conce...   
5695                                          Leonville   

                                           pp_id  
4485                        Diocese of Charlotte  
4091                           St. Joseph Church  
5653                             KY and Chaplain  
2293  joseph-thibaudeau-diocese-of-duluth-652166  
5695                                        2501

### How many cases of laicization?

In [9]:
df[df['status'] == "Laicized"]

id                     name first_name    last_name    status  \
82      6541          Ronald H Paquin     Ronald       Paquin  Laicized   
85     65985             David Carson      David       Carson  Laicized   
111     6550           Robert P Beale     Robert        Beale  Laicized   
112     6551  Frederick J Cartier Jr.  Frederick  Cartier Jr.  Laicized   
113     6552         Joseph K Coleman     Joseph      Coleman  Laicized   
...      ...                      ...        ...          ...       ...   
6558  652487        Donald Buzanowski     Donald   Buzanowski  Laicized   
6564  652493              John Feeney       John       Feeney  Laicized   
6572  652501           Jerome Koerner     Jerome      Koerner  Laicized   
6584  652513           Thomas Stocker     Thomas      Stocker  Laicized   
6585  652514           Richard Thomas    Richard       Thomas  Laicized   

     birth_date ord_date       assignments  \
82         1942     1973  Associate Pastor   
85          NaN     1984               NaN   
111        1941     1970         Assistant   
112        1936     1963         Assistant   
113        1953     1981  Associate Pastor   
...         ...      ...               ...   
6558        NaN     1968        St. Joseph   
6564        NaN     1952        St. Joseph   
6572        NaN     1960          St. Jude   
6584        NaN     1963        St. Joseph   
6585        NaN     1966          St. Mary   

                                          diocese_id  \
82                                  St Monica Parish   
85                                              2502   
111                                           St Ann   
112                                 St Jerome Parish   
113                                 St Angela Parish   
...                                              ...   
6558  Green Bay (1968-1970); St. John the Evangelist   
6564              Green Bay (1952-1954); Holy Rosary   
6572                    Green Bay (1960); Holy Cross   
6584                  Green Bay (1963); St. Boniface   
6585                 Menasha (1966-1967); Holy Cross   

                                           diocese_name  \
82                                              Methuen   
85                              Archdiocese of New York   
111                                    West Bridgewater   
112                                      North Weymouth   
113                                            Mattapan   
...                                                 ...   
6558                    Green Bay (1970-1974); St. Jude   
6564                        Kewaunee (1954); St. Joseph   
6572            Kaukauna (1960-1965); St. Margaret Mary   
6584  De Pere (1963-1971); Diocesan Marriage Tribuna...   
6585                     Kaukauna (1967-1972); St. Mary   

                                              list_name  \
82                05/01/73 - 06/01/81; Associate Pastor   
85    The following archdiocesan clergy have been cr...   
111               06/09/70 - 06/02/75; Associate Pastor   
112                      02/12/63 - 06/06/66; Assistant   
113                       06/30/81 - 04/15/85; Chaplain   
...                                                 ...   
6558                 Green Bay (1974-1975); St. Patrick   
6564                 Sturgeon Bay (1954-1955); St. Rose   
6572                       Neenah (1965-1967); Nativity   
6584                   Greenleaf (1978-1988); St. James   
6585           Kaukauna (1972-1976); SS. Peter and Paul   

                                            slug  \
82                    St John the Baptist Parish   
85    david-carson-archdiocese-of-new-york-65985   
111               St Mary of Annunciation Parish   
112                            St Barbara Parish   
113                K of C Dorchester Lower Mills   
...                                          ...   
6558            Green Bay (1975-1982); St. Louis   
6564         Clintonville (1955-1956); St. Peter 

##### = When a priest is laicized, he is dismissed from a clerical state and secularized, becoming a "layperson." There were 349 total cases.

### How many laicized in Los Angeles?

In [10]:
df[(df['status'] == "Laicized") &  (df['diocese_id'] == "Archdiocese of Los Angeles")] 

Empty DataFrame
Columns: [id, name, first_name, last_name, status, birth_date, ord_date, assignments, diocese_id, diocese_name, list_name, slug, pp_id]
Index: []

##### = 0?

### How many incarcerated?

In [11]:
df[df['status'] == "Incarcerated"]

id                   name first_name        last_name        status  \
1231  651527           Gary Mercure       Gary          Mercure  Incarcerated   
1754   70409   William Claude Casey    William     Claude Casey  Incarcerated   
1810  651715   Jose Vazquez Morales       Jose  Vazquez Morales  Incarcerated   
2214  652091               Rudy Kos       Rudy              Kos  Incarcerated   
3423  653648          Wayland Brown    Wayland            Brown  Incarcerated   
3713  654190           James Talbot      James           Talbot  Incarcerated   
4044   70410  Edward Joseph McKeown     Edward   Joseph McKeown  Incarcerated   
6051  651590     Joseph D. Maurizio     Joseph         Maurizio  Incarcerated   
6424  652332           Clint Landry      Clint           Landry  Incarcerated   
6441  652365            James Blume      James            Blume  Incarcerated   
6449  652374            Paul LeBrun       Paul           LeBrun  Incarcerated   

     birth_date ord_date                                        assignments  \
1231        NaN     1974                                                NaN   
1754       1934     1969  Associate Pastor of Sts. Peter and Paul Church...   
1810        NaN     2008             Parochial Vicar: Immaculate Conception   
2214       1945     1981  All Saints ; St. John Nepomucene (Ennis); St. ...   
3423        NaN     1977                                                NaN   
3713        NaN      NaN                               St. George's College   
4044       1944     1970  Associate Pastor of St. Edward Church in Nashv...   
6051        NaN      NaN                                                NaN   
6424        NaN      NaN                                                NaN   
6441        NaN     1980                                             July 7   
6449        NaN     1983                                        St. Therese   

                                             diocese_id  \
1231                                               2522   
1754                                               2607   
1810  Laurel (1/10/09 – 7/1/11); Parochial Vicar: ...   
2214                                               2556   
3423                                               2640   
3713                                           Kingston   
4044  St. Ann Church in Deer Lodge and St. Thomas th...   
6051                                               2525   
6424                                               2564   
6441               1980 – St. Anthony de Padua Parish   
6449                               Little Flower Parish   

                                           diocese_name  \
1231                                  Diocese of Albany   
1754                               Diocese of Nashville   
1810          Hattiesburg; (7/1/11 – 1/15/14); Pastor   
2214                                  Diocese of Dallas   
3423                                Diocese of Savannah   
3713                                            Jamaica   
4044  in residence at St. Augustine Church in Signal...   
6051                       Diocese of Altoona-Johnstown   
6424                               Diocese of Fairbanks   
6441                                         South Bend   
6449                                         South Bend   

                                              list_name  \
1231  CLERGY CREDIBLY ACCUSED WHILE SERVING IN THE D...   
1754  Priests who served in the diocese who have bee...   
1810                                 St. Francis Xavier   
2214  LIST OF ALL PRIESTS WITH CREDIBLE ALLEGATIONS ...   
3423  DIOCESAN PRIESTS WITH CREDIBLE ALLEGATIONS OF ...   
3713     1962 - 1964 (education); Immaculate Conception   
4044  then Director of the Chattanooga Deanery Catho...   
6051  The list below identifies Diocesan priests who...   
6424                                            Priests   
6441                                    Indiana; July 7   
6449                 

##### = Only 11 incarcerations? 

### Status of clergymembers in each diocese? 

In [12]:
df.groupby(['status'])['diocese_name'].count()

status
1968: Left the Jesuits 1999: Died    1
1992: Left the Jesuits 2004: Died    1
1993: Died                           1
1994: Removed from ministry          1
1995: Died                           1
                                    ..
retired from ministry                2
revoked                              3
suspended                            2
suspended from ministry              8
voluntarily returned to lay state    1
Name: diocese_name, Length: 1161, dtype: int64

Findings: This dataset shows that there were 6,753 unique counts of credible sexual assault accusations directed at clergymembers from 2002 to 2020. That's more than 375 cases a year. With 365 days in a year, this means that at least one sexual assault took place every single day. 

I also learned that Robert Poandl had the highest count of sexual assault combined: 10. 2,576 of the accused clergymembers are deceased, which is the highest percentage of the current status of these accused clergymembers. Most of them held assistant pastor positions, and 301 of the assaults occurred at the Archdiocese of Los Angeles. 

I was surprised to see that only 11 counts of incarceration came up in the data, which seems so overwhelmingly miniscule when compared to the sheer number of cases. 